actors/builtin/miner/miner_state.go

```diff
@@ -798,6 +798,7 @@ func (st *State) AddLockedFunds(store adt.Store, currEpoch abi.ChainEpoch, vesti
 	if err != nil {
 		return err
 	}
+	st.LockedFunds = vestingSum
```

Comment:
we need to set this value else `Unlock*` methods will [always](https://github.com/filecoin-project/specs-actors/pull/311/files#diff-2580e4b82238370ee854a52e8d5428e7R852) [abort](https://github.com/filecoin-project/specs-actors/pull/311/files#diff-2580e4b82238370ee854a52e8d5428e7R896)